### DISCUSSION TITLE
Extra move idea

### DISCUSSION THREAD

**Discussion by torchdeveloper** [BeeeBea/FastMove#27] [Ideas]
When you jump right as you start a slide, your character performs what almost looks like a flying kick. It would be sick if this could actually hurt enemies, dealing a small amount of damage but a good chunk of knockback, allowing you to send mobs flying into lava pools and whatnot at the risk of sending yourself in too if you're not careful.

https://github.com/BeeeBea/FastMove/assets/90207712/0a5b1e8c-798f-4439-a36b-778df9ac0c34

**Comment by Saiko125** [comment]
That idea reminds me of what epic added to fortnite when they added new movement except there it only deals knockback iird
